src/planning/test/plan/planificator-test.ts

```diff
@@ -169,7 +169,7 @@ describe('remote planificator', () => {
       await verifyConsumerResults(5);
     });
   });
-  it(`merges remotely produced suggestions`, async () => {
+  it.skip(`merges remotely produced suggestions`, async () => {
```

Comment:
file an issue on Maria to unskip this when we can work out what the timing issues are.

Reply:
Yup, already did that.